packages/waku/src/lib/middleware/rsc.ts

```diff
@@ -1,7 +1,7 @@
 import path from 'node:path';
 import fsPromises from 'node:fs/promises';
 import type { IncomingMessage, ServerResponse } from 'node:http';
-import type { ViteDevServer } from 'vite';
+import { resolveConfig as resolveViteConfig, mergeConfig, type ViteDevServer } from 'vite';
```

Comment:
Just stylistic preference.
```suggestion
import { resolveConfig as resolveViteConfig, mergeConfig } from 'vite';
import type { ViteDevServer } from 'vite';
```

Reply:
Actually, `rsc` middleware should work without vite in production.
So, these vite function should be imported dynamically `getViteServer`.